pkg/scheduler/watchers/membercluster/controller_integration_test.go

```diff
@@ -512,19 +513,31 @@ var _ = Describe("scheduler member cluster source controller", Serial, Ordered,
 		})
 	})
 
-	Context("ready cluster has left", func() {
+	FContext("ready cluster is leaving should not trigger any reschedule but left should", func() {
```

Comment:
Just a nit: the name reads a bit confusing to me; how about `leaving clusters should not trigger rescheduling until the member cluster object is fully deleted`?